models/Result.js

```diff
@@ -31,21 +31,39 @@ const resultsSort = (a, b) => { // eslint-disable-line consistent-return, array-
   if (regularVersusRegular(a) && regularVersusRegular(b)) return 0;
 };
 
-module.exports = class Results {
+class Result {
+
+  constructor(id, fC, fNN, nA, opponentName, pending, perfDate, perfName, perfId, sC, sNN, spotId, uOA, uTA, winnerId) {
+    this._id = id;
+    this._firstComments = fC;
+    this._firstNameNumber = fNN;
+    this._needsApproval = nA;
+    this._opponentName = opponentName;
+    this._pending = pending;
+    this._performanceDate = new Date(perfDate);
+    this._performanceName = perfName;
+    this._performanceId = perfId;
+    this._secondComments = sC;
+    this._secondNameNumber = sNN;
+    this._spotId = spotId;
+    this._userOneAlternate = uOA;
+    this._userTwoAlternate = uTA;
+    this._winnerId = winnerId;
+  }
 
-  static getAttributes() {
+  static get attributes() {
     return modelAttributes;
   }
 
-  static getIdName() {
+  static get idName() {
     return 'id';
   }
 
-  static getTableName() {
+  static get tableName() {
     return 'results';
   }
 
-  approve(ids) {
+  static approve(ids) {
```

Comment:
You can disregard my comment from earlier. Having these static methods to mutate isn't something I've often seen, but you're consistent, so 👌